ext-src/models/IqComponentModel.ts

```diff
@@ -50,62 +50,84 @@ export class IqComponentModel implements ComponentModel {
       return new Promise((c, e) => "my stubbed content entry");
     }
   
-    public async evaluateComponents() {
+    public evaluateComponents(): Promise<any> {
```

Comment:
We technically didn't need to mark any of these as async, since we have a promise being returned higher up. I don't really like how many chains deep we have it right now (we can likely simplify quite a bit of this), but may as well remove these until we have an actual need to use `await` in the function. I mark the methods as `Promise<any>` so someone can at least see what's going on :)